recipes/fastdtw/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "fastdtw" %}
+{% set version = "0.3.2" %}
+{% set sha256 = "09a24d62a8eb72f87c03692dffae4b8d8499b53f503e9abce5e717c5b705dff9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # noarch: python
```

Comment:
As you are using ``cython`` this can't be ``noarch`` indeed. So, please remove this commented section